views/pages/cartoons/team.pug

```diff
@@ -0,0 +1,25 @@
+extends /views/templates/frontend-base.pug
+
+append styles
+  link(href="/assets/css/pages/cartoons/team.css" rel="stylesheet")
+
+append scripts
+  script(src="/assets/js/collapsible.js" defer)
+
+
+block body
+
+    +section("white")
+        h1 Cartoons Team
+
+        each term in data.terms
+            button(type="button" class="collapsible-button" value="" + title + "")= term.term
+            div(class="collapsible-content")
+                div(class="collapsible-cartoon-content")
```

Comment:
I wonder if collapsibles are the right approach here? From a user's standpoint, I think it's probably easier for them to just be able to look through a long page of images rather than click open each one.

thoughts?

Reply:
tbh i thought the original one-pager was kinda long-- if it gets updated every term i feel like there's a point where scrolling is just annoying

alternatively i can do a carousel at the top (before the team names + roles) that just goes through the cartoons for each year?

Reply:
if you insist on the one-pager i'm also ok to just do that, but let me know

Reply:
i'll trust your judgement!